server/game/cards/events/02/trialbycombat.js

```diff
@@ -0,0 +1,36 @@
+const DrawCard = require('../../../drawcard.js');
+const ApplyClaim = require('../../../gamesteps/challenge/applyclaim.js');
+
+class TrialByCombat extends DrawCard {
```

Comment:
Looks like you've inadvertently included commits from your previous PR. We typically use a feature-branch + squash-merge flow.